pom.xml

```diff
@@ -12,8 +12,8 @@
     <version>1.9.5</version>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
Not sure we want to do this now already .. we could just make it work with 17 to build .. and then switch language level later

Reply:
Sounds good. I closed this and will add the build options in once the maven wrapper PR https://github.com/trinodb/trino-gateway/pull/8 is merged. I don't think this should conflict with Vishal's package renaming.

Reply:
Yes, that makes sense. I will have 2 more PR to bring our code in sync with this repo and then we can go on enhancing the rest.